### DISCUSSION TITLE
Implements OTEL Metrics based on standard dotnet on system diagnostics

### DISCUSSION THREAD

**Discussion by aygalinc** [rabbitmq/rabbitmq-dotnet-client#1733] [General]
### Is your feature request related to a problem? Please describe.

Acutally there is no metrics provide by the client and it s up to the all apps to implement them if needed. 

### Describe the solution you'd like

Implement metrics through standard system diagnostic Meter that can be easily integrates in OTEL or Prometheus library.

**Comment by aygalinc** [comment]
I wil do a PR when i got some time

**Comment by michaelklishin** [comment]
@aygalinc your issues are repeatedly deleted and moved to discussions because they are not specific enough for our team.

You are welcome to submit a PR that demonstrates the idea but "OTEL metrics based on standard .NET system diagnostics" is a really vague definition.
